lib/ansible/plugins/lookup/consul_kv.py

```diff
@@ -5,6 +5,10 @@
 
 __metaclass__ = type
 
+ANSIBLE_METADATA = {'metadata_version': '1.1',
```

Comment:
Not sure if this is required or not in plugin. cc @bcoca

Reply:
I asked that in IRC yesterday and @gundalow asked me to add it as they are planning to implement the check for ansible metadata for plugins, too.

Reply:
I am sorry, I was not aware of this thing in pipeline.